distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -231,25 +235,78 @@ type Table
        > Example
          Remove columns matching a regular expression.
 
-             table.remove_columns [Column_Selector.By_Name "foo.+" Case_Sensitivity.Insensitive use_regex=True]
+             table.remove_columns "foo.+".to_regex Case_Sensitivity.Insensitive
 
        > Example
          Remove the first two columns and the last column.
 
              table.remove_columns [-1, 0, 1]
     @columns Widget_Helpers.make_column_name_vector_selector
-    remove_columns :  Vector (Integer | Text | Column_Selector) | Text | Integer -> Boolean -> Problem_Behavior -> Table ! No_Output_Columns | Missing_Input_Columns
-    remove_columns self (columns = [self.columns.first.name]) (error_on_missing_columns = False) (on_problems = Report_Warning) =
-        new_columns = self.columns_helper.remove_columns selectors=columns error_on_missing_columns=error_on_missing_columns on_problems=on_problems
+    remove_columns :  Vector (Integer | Text | Regex) | Text | Integer -> Regex -> Boolean -> Problem_Behavior -> Table ! No_Output_Columns | Missing_Input_Columns
+    remove_columns self (columns = [self.columns.first.name]) (case_sensitivity=Case_Sensitivity.Default) (error_on_missing_columns:Boolean=False) (on_problems:Problem_Behavior=Report_Warning) =
+        new_columns = self.columns_helper.remove_columns columns case_sensitivity error_on_missing_columns=error_on_missing_columns on_problems=on_problems
         self.updated_columns new_columns
 
+    ## ALIAS select_na
+       ALIAS select_missing_columns
+
+       Select columns which are either all blank or contain blank values. If no
+       rows are present, all columns are considered blank.
+
+       Arguments:
+       - when_any: By default, only columns consisting of all blank cells are
+         selected. If set to `True`, columns with one or more blank values are
+         selected.
+       - treat_nans_as_blank: If `True`, then `Number.nan` is considered as
+         blank.
+
+       ? Blank values
+         Blank values are `Nothing`, `""` and depending on setting `Number.nan`.
```

Comment:
I feel like I'd sometimes want to distinguish between a `""` and `Nothing` - depending on context these may have some different meaning - e.g. in database `NULL` vs empty string are different entries and they can mean different things.

Do you think we should add `treat_empty_string_as_blank` like `treat_nan_as_blank`?

I'm not strongly attached to it since this can be emulated relatively easily using more primitive operations. So whatever you think will be better.

Reply:
This was be design when we agreed the previous API design for removing blank values. 

I appreciate there is a difference but the single value case can easily be done using the filter API.